test/test_puma_server.rb

```diff
@@ -1,9 +1,10 @@
 require_relative "helper"
 
 class TestPumaServer < Minitest::Test
+  parallelize_me!
 
   def setup
-    @port = 0
+    @port = UniquePort.call
```

Comment:
Why can't we just use port = 0 everywhere, I wonder?

Reply:
Good question.  Only thing is that all the code would have to be changed for the 'clients', but we could set that in `wait_for_server_to_boot`?

I'm !AFK now, so as soon as the CI finishes, I'll have a look, as it's pretty 'fresh' in my mind...